commons/admin-console/custom-domain-names.md

```diff
@@ -11,80 +11,131 @@ keywords:
+### Your Application Runs in the Europe/Paris ('PAR') Zone
 
-    <a href="http://www.gandi.net" target="_blank">Gandi.net</a> example :
-    <figure class="cc-content-img">
-      <img  src="/doc/assets/images/domain2.png">
-    </figure>
-    <figcaption>CNAME records example</figcaption>
+<table class="table table-bordered" style="text-align:center">
+  <tr>
+    <th><center>Record Type</center></th>
+    <th><center>Value</center></th>
+  </tr>
+  <tr>
+    <td>CNAME <div><span class="label label-success">Recommended</span></div></td>
+    <td>``domain.par.clever-cloud.com``</td>
+  </tr>
+  <tr>
+    <td>A<div><small>Only if CNAME is not available</small></div></td>
+    <td>Two records on ``62.210.112.171`` and ``62.210.92.244``</td>
+  </tr>
+</table>
 
-  * Domains like ``example.com``
+<br/>
 
-    On your domain name provider, point two **A records** to ``62.210.112.171`` and ``62.210.92.244``
+### Your Application Runs in the North-America/Montreal ('MTL') Zone
 
+<table class="table table-bordered" style="text-align:center">
+  <tr>
+    <th><center>Record Type</center></th>
+    <th><center>Value</center></th>
+  </tr>
+  <tr>
+    <td>CNAME <div><span class="label label-success">Recommended</span></div></td>
+    <td>``domain.mtl.clever-cloud.com``</td>
+  </tr>
+  <tr>
+    <td>A<div><small>Only if CNAME is not available</small></div></td>
+    <td>Two records on ``68.71.34.21`` and ``68.71.34.20``</td>
+  </tr>
+</table>
 
-### Your application runs in the North-America ('MTL') zone
 
-You have two non exclusive choices to add custom personal domain names:
+<br/>
 
-  * Sub-domains like ``hello.example.com`` and wildcard sub-domains like ``*.example.com``
+Two kinds of records are supported, Type A and CNAME. **The use of a CNAME record is highly recommended.** The CNAME configuration will ensure that your DNS configuration will be up-to-date whereas A records will require an action for you if the IP adresses have to be changed. We monitor all domain names linked to apps, so obsolete or incorrect configurations will be notified by email by our support team to apps owners.
 
-    On your domain name provider, point the associated *hello* **CNAME record** to ``domain.mtl.clever-cloud.com``
 
-    <a href="http://www.gandi.net" target="_blank">Gandi.net</a> example :
-    <figure class="cc-content-img">
-      <img  src="/doc/assets/images/domain2.png">
-    </figure>
-    <figcaption>CNAME records example</figcaption>
+<div class="panel panel-warning">
+  <div class="panel-heading">
+     <h4>Warning on CNAME Availability</h4>
+  </div>
+  <div class="panel-body">
+    You cannot use a CNAME  on a top-level domain, or on a subdomain which already has DNS records.
+  </div>
+</div>
 
-  * Domains like ``example.com``
+If you want to redirect your top-level domain name to a subdomain through HTTP (eg ``domain.com`` to ``www.domain.com``), most domain name registrars provide a **web redirection service**.
```

Comment:
If you want to make you application available from a domain name which does not support CNAME records (eg ``example.com`` in addition to ``www.example.com``, check if your registrar provides a **web redirection service**. This way, you only have to make ``ẁww.example.com`` point to Clever Cloud. Please note that web redirection provided registrars only work over HTTP.